### PR TITLE
Updating image version to 311.153.20191113 and create 12.1

### PR DESCRIPTION
**Pull request by kwoodson**
```release-note
Updating image version to 311.153.20191113 and create 12.1
```

### CODE REVIEW

**Comment by jim-minter** [comment]
also unit test failures

**Comment by kwoodson** [comment]
@jim-minter @asalkeld PTAL

**Comment by kwoodson** [comment]
quota exceeded
```
 Maximum allowed: 300, Current in use: 300, Additional requested: 2.
```

File: pluginconfig/pluginconfig-311.yaml [hunk 1]
```diff
@@ -302,6 +302,48 @@ versions:
       sync: osarpint.azurecr.io/openshift-on-azure/azure:v12.0
       tlsProxy: osarpint.azurecr.io/openshift-on-azure/azure:v12.0
       logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod10182019
+  v12.1:
```
Comment (jim-minter) [inline pluginconfig/pluginconfig-311.yaml]: @kwoodson you need to set `pluginVersion: v12.1` at the top of this file, otherwise lgtm

Comment (kwoodson) [reply pluginconfig/pluginconfig-311.yaml]: Updated.  Thanks

**Comment by jim-minter** [comment]
I am expecting this to fail tests until @asalkeld has built the images involved

/lgtm

**Comment by kwoodson** [comment]
It seems to be failing now on the tests since we moved the pluginversion.  It is requesting hashes for the tests on the previous v12.0.  Updating with those hashes.

**Pull request merged**